qemu/qemu_s390x.go

```diff
@@ -0,0 +1,121 @@
+// +build s390x
+
+/*
+// Copyright (c) 2016 Intel Corporation
```

Comment:
IBM ?

Reply:
Year should be (or include) 2018 too.

Reply:
The copyright is with me... changed the files accordingly